lib/src/helper/helper_functions.dart

```diff
@@ -0,0 +1,4 @@
+import '../utils/constants.dart';
+
+bool isImageUrl(String message) =>
```

Comment:
what if I messaged use jpg will it work ?

Reply:
Yes